app/src/pages/trace/SessionDetailsTraceList.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useMemo } from "react";
+
+import { Flex, Icon, Icons, Text, View } from "@arizeai/components";
+
+import { Link } from "@phoenix/components";
+import {
+  AnnotationLabel,
+  AnnotationTooltip,
+} from "@phoenix/components/annotation";
+import { LatencyText } from "@phoenix/components/trace/LatencyText";
+import { TokenCount } from "@phoenix/components/trace/TokenCount";
+import { fullTimeFormatter } from "@phoenix/utils/timeFormatUtils";
+
+import { SessionDetailsQuery$data } from "./__generated__/SessionDetailsQuery.graphql";
+import { EditSpanAnnotationsButton } from "./EditSpanAnnotationsButton";
+
+function RootSpanMessage({
+  role,
+  value,
+}: {
+  role: "HUMAN" | "AI";
+  value: string;
+}) {
+  return (
+    <View
+      alignSelf={role === "HUMAN" ? "start" : "end"}
+      borderRadius={"medium"}
+      borderColor={"dark"}
+      borderWidth={"thin"}
+      padding={"size-100"}
+      maxWidth={"70%"}
+    >
+      <Flex direction={"column"} gap={"size-100"}>
+        <Text color="text-700" textSize="medium">
+          {role}
+        </Text>
+        <Text>{value}</Text>
+      </Flex>
+    </View>
+  );
+}
+
+type SessionTraceRootSpan = NonNullable<
+  NonNullable<
+    SessionDetailsQuery$data["session"]["traces"]
+  >["edges"][number]["trace"]["rootSpan"]
+>;
+
+type RootSpanProps = {
+  rootSpan: SessionTraceRootSpan;
+};
+
+function RootSpanDetails({
+  rootSpan,
+  user,
+}: RootSpanProps & { user?: string | null }) {
+  const startDate = useMemo(() => {
+    return new Date(rootSpan.startTime);
+  }, [rootSpan.startTime]);
+  return (
+    <View height={"100%"}>
+      <Flex
+        direction={"column"}
+        justifyContent={"space-between"}
+        height={"100%"}
+      >
+        <Flex direction={"column"} gap={"size-200"}>
+          <Flex direction={"row"} justifyContent={"space-between"}>
+            <Text>Trace ID: {rootSpan.context.traceId}</Text>
+            <Link
+              to={`/projects/${rootSpan.project.id}/traces/${rootSpan.context.traceId}`}
```

Comment:
maybe also pin the span, otherwise the selection is kind of random
```suggestion
              to={`/projects/${rootSpan.project.id}/traces/${rootSpan.context.traceId}?selectedSpanNodeId=${rootSpan.id}`}
```

Reply:
oh i thought it just chose the root span if there wasn't one, will do